ansible/collectory-standalone.yml

```diff
@@ -1,8 +1,8 @@
 - hosts: collectory
   roles:
-    - common
-    - java    
-    - mysql
-    - tomcat    
-    - apache
-    - collectory
+    - { role: common, collectory: ala-collectory, collectory_version: 1.0 }
+    - { role: java, collectory: ala-collectory, collectory_version: 1.0 }
+    - { role: mysql, collectory: ala-collectory, collectory_version: 1.0 }
+    - { role: tomcat, collectory: ala-collectory, collectory_version: 1.0 }
+    - { role: apache, collectory: ala-collectory, collectory_version: 1.0 }
+    - { role: collectory, collectory: ala-collectory, collectory_version: 1.0 }
```

Comment:
Every task (common, java, mysql, etc.) includes setfacts.yml that need to set the war url, which contains the application Id and Version. Our application Id and Version are variables. When being deployed, any task will stop working with the following error example if we did not pass the parameters:

TASK: [java | set remote repositories (ALA)] ********************************** 
fatal: [vagrant1] => One or more undefined variables: 'collectory' is undefined
FATAL: all hosts have already failed -- aborting

That is the reason I pass the parameters for all tasks.